### DISCUSSION TITLE
Why Alpine for Docker and not Ubuntu

### DISCUSSION THREAD

**Discussion by uhlhosting** [anonaddy/docker#248] [Q&A]
Hi,

I have noticed the image is built against Alpine, any docker pro user would know NOT to do so, since alpine it has many benefits, yet not for such a complex/big app as anonnaddy. 

It creates more problems than it solves. And this after already checking the huge discrepancies between docker / self hosted . Plus the limitations of Alpine to run apps with several services, so was never intended for that.

Please consider moving the image to debian/Ubuntu based OS for the better management and to be at least a bit aligned to the proper microservices usage.

**Comment by crazy-max** [comment]
> any docker pro user would know NOT to do so

Why?

> It creates more problems than it solves. And this after already checking the huge discrepancies between docker / self hosted . Plus the limitations of Alpine to run apps with several services, so was never intended for that.

What issues?

> Please consider moving the image to debian/Ubuntu based OS for the better management and to be at least a bit aligned to the proper microservices usage.

I'm all in if you have real arguments
